apps/src/lib/ui/PopUpMenu.jsx

```diff
@@ -71,10 +71,11 @@ export default class PopUpMenu extends Component {
           targetPoint={this.props.targetPoint}
           offset={this.props.offset}
           className={this.props.className}
-          children={this.props.children}
           showTail={this.props.showTail}
           style={this.props.style}
-        />
+        >
+          {this.props.children}
+        </MenuBubble>
```

Comment:
Intentional part of this change?

Reply:
Yep! One of the new default rules is [`react/no-children-prop`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-children-prop.md).  We only had two violations of that rule in our code and it's super-easy to fix, so I figured I'd fix instead of disabling.

Reply:
👍